activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/BPMNSequenceFlowTakenEventHandler.java

```diff
@@ -83,11 +72,11 @@ private BPMNSequenceFlowEntity createBpmnSequenceFlowEntity(CloudRuntimeEvent<?,
             event.getAppName(),
             event.getAppVersion()
         );
-        bpmnSequenceFlowEntity.setId(UUID.randomUUID().toString());
+        bpmnSequenceFlowEntity.setId(entityId);
         bpmnSequenceFlowEntity.setElementId(bpmnSequenceFlow.getElementId());
         bpmnSequenceFlowEntity.setProcessDefinitionId(bpmnSequenceFlow.getProcessDefinitionId());
         bpmnSequenceFlowEntity.setProcessInstanceId(bpmnSequenceFlow.getProcessInstanceId());
-        bpmnSequenceFlowEntity.setDate(new Date(activityEvent.getTimestamp()));
+        bpmnSequenceFlowEntity.setDate(new Date(event.getTimestamp()));
```

Comment:
Not sure is correct change the date they can be similar but are actually different, WDYT @erdemedeiros ?

Reply:
Hmm, actually it's not changing the date, it's setting the date for the `bpmnSequenceFlowEntity` created at the line `68` and the value used to set it is the same as before (the variable only get renamed from `activityEvent ` to `event` as part of method extraction.